File: program72.java
//accept n numbers and perform addition of that numbers
import java.lang.*;
import java.util.*;

class ArrayR
{
    public int AddNumbers(int []array)
    {
        int sum=0;
        for(int i=0;i<array.length;i++)
        {
            sum=sum+array[i];
            
        }
        return sum;
    }
}

class program72
{
    public static void main(String arg[])
    {
        int ret=0;
        Scanner sobj=new Scanner(System.in);
        System.out.println("Enter the size of the Array");
        int isize=sobj.nextInt();

        int []MyArray=new int[isize];

        System.out.println("Enter the elements of the Array");
        for(int i=0;i<isize;i++)
        {
            MyArray[i]=sobj.nextInt();
        }
        ArrayR obj=new ArrayR();
        System.out.println("Addition of numbers is :"+obj.AddNumbers(MyArray));
    }
}

File: program72.c
// accept n numbers and perform addition of that number
#include<stdio.h>
#include<stdlib.h>

int AddNumbers(int Arr[],int iLength)
{
    int iSum=0,iCnt=0;
    for(iCnt=0;iCnt<iLength;iCnt++)
    {
        iSum=iSum+Arr[iCnt];
    }
    return iSum;

}

int main()
{
    int *ptr=NULL;
    int iCnt=0,iRet=0;
    int iSize=0;

    printf("ENter number of elements\n");
    scanf("%d",&iSize);

    ptr=(int*)malloc(iSize*sizeof(int));
    printf("Enter the values ");

    for(iCnt=0;iCnt<iSize;iCnt++)
    {
        scanf("%d",&ptr[iCnt]);
    }
    iRet=AddNumbers(ptr,iSize);

    printf("Addition is %d\n",iRet);
    free(ptr);

    return 0;
}